src/cpu/flags.cpp

```diff
@@ -1051,7 +1051,11 @@ void FillFlagsNoCFOF(void) {
 template <typename T>
 void set_cpu_test_flags_for_division(const T quotient) noexcept
 {
-	SETFLAGBIT(OF, false);
+	const CpuTestFlags cpu_test_flags = {
+	        .HasOverflow = false,
+	};
```

Comment:
Won't this generate a warning that not all members have been initialised? If so, maybe don't use the designated initialiser like this.

Reply:
(I wasn't seeing warnings on my side or any new ones in CI) but I wound up reworking how this CpuTestFlags object works.

Previously all of the test flags were cleared (which was fine for the DIV instructions), but other instructions only clear and set a subset. So the new implementation lets you define the set of flags you'll be adjusting at construction.